test/api/getFee/index.ts

```diff
@@ -19,12 +19,10 @@ export default <TestSuite>{
   },
 
   'getFee - high load_factor': async (api, address) => {
-    api.connection._send(
```

Comment:
This is the main test change in this PR: `_send` (private) -> `request` (public).